.test-infra/jenkins/jira_utils/jira_manager.py

```diff
@@ -168,21 +174,42 @@ def _append_descriptions(self, issue, dep_name, dep_latest_version):
     logging.info("Updating JIRA issue {0} to track {1} upgrade process".format(
         issue.key,
         dep_name))
-    description = issue.fields.description + """\n\n{0}\n
-        Please review and upgrade the {1} to the latest version {2} \n 
+    description = self._create_descriptions(dep_name, dep_current_version, dep_latest_version, issue=issue)
+    try:
+      self.jira.update_issue(issue, description=description)
+    except Exception as e:
+      traceback.print_exc()
+      logging.error("Failed updating issue: "+ str(e))
+
+
+  def _create_descriptions(self, dep_name, dep_current_version, dep_latest_version, issue = None):
```

Comment:
Put the code of description creating into an independent method, so that it could be reused by _create_issue and _append_descriptions.